Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM ubuntu:20.04
-MAINTAINER Stanislav Bogatyrev <stanislav@nspcc.ru>
```

Comment:
I'd replace it with `NeoSPCC <info@nspcc.io>`

Reply:
done